frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -462,6 +523,32 @@ function ReaderHighlight:addToMainMenu(menu_items)
             })
         end,
     })
+    table.insert(menu_items.highlight_options.sub_item_table, {
+        text = _("Apply default style to all highlights"),
+        callback = function(touchmenu_instance)
+            UIManager:show(ConfirmBox:new{
+                text = _("Are you sure you want to edit all highlights."),
+                icon = "texture-box",
+                ok_callback = function()
+                    local count = 0
+                    for _, items in pairs(self.view.highlight.saved) do
+                        if items then
+                            count = count + #items
+                            for i = 1, #items do
+                                local item = items[i]
+                                item.drawer = self.view.highlight.saved_drawer
+                                item.color = self.view.highlight.saved_color
+                            end
+                        end
+                    end
+                    UIManager:setDirty(self.dialog, "ui")
+                    UIManager:show(Notification:new{
+                        text = T(_("Applied default style to %1 highlights"), count),
```

Comment:
We use N_ here for plurals different forms
https://github.com/koreader/koreader/blob/873503369c21f3ddc265c0623b477ceffd521f88/frontend/apps/reader/modules/readerbookmark.lua#L613-L617